backend/src/main/java/com/jos/spotifyclone/controller/playlistsOfCurrentUser.java

```diff
@@ -0,0 +1,29 @@
+package com.jos.spotifyclone.controller;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.jos.spotifyclone.services.SpotifyConnect;
+import com.wrapper.spotify.exceptions.SpotifyWebApiException;
+import com.wrapper.spotify.model_objects.specification.Paging;
+import com.wrapper.spotify.model_objects.specification.PlaylistSimplified;
+import org.apache.hc.core5.http.ParseException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+@RequestMapping("/user/playlists")
+@RestController
+public class playlistsOfCurrentUser {
```

Comment:
As this is a controller - it is standard practice to have controller in the name and Java naming convertion is camcelcase with the first character uppercase on classes.

Change this to `PlaylistController` please